packages/gestalt/src/Sheet.js

```diff
@@ -19,27 +43,52 @@ import Row from './Row.js';
 import StopScrollBehavior from './behaviors/StopScrollBehavior.js';
 import sheetStyles from './Sheet.css';
 import TrapFocusBehavior from './behaviors/TrapFocusBehavior.js';
-import useReducedMotion from './useReducedMotion.js';
 
-type AnimationType = 'in' | 'out';
-
-type Props = {|
+type SheetMainProps = {|
   accessibilityDismissButtonLabel: string,
   accessibilitySheetLabel: string,
-  children?: Node,
+  children: Node,
   closeOnOutsideClick?: boolean,
   footer?: Node,
-  heading?: string,
   onDismiss: () => void,
   size?: 'sm' | 'md' | 'lg',
 |};
 
+type SheetProps = {|
+  ...SheetMainProps,
+  heading?: string,
```

Comment:
Should `heading` be required? Would we ever want to display a Sheet without a heading?

Reply:
The initial figma design that I received has a case without a heading, that's why I did it like that. Will share you the file on Slack.